README.md

```diff
@@ -132,6 +156,13 @@ smooth_fq_linear_to_inference(model)
 model = torch.compile(model, mode='max-autotune')
 model(input)
 ```
+## Success stories
```

Comment:
I'd move these up near the top they're more important than features and a lot of people know the fast series already